docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -611,3 +611,84 @@ resolve to a `Map`.
 
 See: link:https://github.com/apache/tinkerpop/tree/master/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/MergeVertexStep.java[source],
 link:https://tinkerpop.apache.org/docs/current/reference/#mergev-step[reference]
+
+=== call()
```

Comment:
i know `element()` was sort of a secondary thing here, but for completeness sake, could you add a section for that here as well?